bauble_macro_util/src/lib.rs

```diff
@@ -676,12 +678,27 @@ pub fn derive_bauble_derive_input(
 
                     Ok(())
                 }
+                "always_ref" => {
+                    if flatten {
+                        Err(meta.error("duplicate `always_ref` attribute"))?
+                    }
```

Comment:
Does this mean that flatten implies always_ref? If so, the error message should mention that.

Reply:
I've just forgotten to change it to `alway_ref`, but that's a good point, flatten and always_ref are incompatible so should error there too.